src/weppy-ember.ts

```diff
@@ -0,0 +1,409 @@
+module WeppyEmber {
+	export interface MetricsReport {
+		startTime: number;
+		duration: number;
+		url: string;
+		klass?: string;
+		method?: string;
+		pattern?: string;
+	}
+
+	export interface WeppyEmberConfig {
+		enableAjaxFilter: boolean;
+		enableLogging: boolean;
+		error?: (message: string) => void;
+		log?: (message: string) => void;
+		minDuration: number;
+		send: (events: any[], metrics: MetricsReport) => void;
+		warn?: (message: string ) => void;
+	}
+
+	export var VERSION: string = '0.1.0';
+	export var loaded: boolean = false;
+	export var enabled: boolean = false;
+	export var config: WeppyEmberConfig = {
+		enableAjaxFilter: false,
+		enableLogging: true,
+		minDuration: 50,
+		log: (message) => { console.log(message); },
+		send: (events, metrics) => { console.log('Sending (Dry Run)\n============================\n', metrics); },
+		warn: (message) => { console.log('warn: ' + message); },
+		error: (message) => { console.log('error: ' + message); }
+	};
+
+	var traceStack = [];
+
+	function getLastTraceItem () {
```

Comment:
I don't recall offhand our standard spacing for functions, but the spacing between function names and parentheses is inconsistent for these functions.